en/core-libraries/time.rst

```diff
@@ -118,6 +118,9 @@ It is also allowed to directly assign those properties to modify the date::
 
     $time->year = 2015;
     $time->timezone = 'Europe/Paris';
+    
+.. note::
+    Starting with version 3.4, $now->timezone returned the object(DateTimeZone) 
```

Comment:
not related to cake 
this is Chronos  feature 
```php
    $now = Time::now();
    debug($now->timezone);
    $now = new \Cake\Chronos\Chronos();
    debug($now->timezone);
```
you should remove mention to cake

Reply:
Cake uses Chronos internally

Reply:
I need to leave only:

.. note::
    $now->timezone returned the object(DateTimeZone) (Cake uses Chronos internally)

or remove request ?